user-docs/pupil-player.md

```diff
@@ -149,15 +149,58 @@ This plugin can be used to overlay the eye video on top of the world video. Note
   + `show` - show or hide eye video overlays.
   + `horiz. and vert. flip` - flip eye videos vertically or horizontally
 
-### Analysis Plugins
+### Data Source Plugins
 
-These plugins are simple unique plugins, that operate on the gaze data for analysis and visualizations.
+Data source plugins provide data of a specific topic -- currently `pupil` and `gaze` positions. Plugins that provide data for a specific topic are complementary to each other, e.g. `Pupil Positions From Recording` and `Offline Pupil Detector` (see below). Complementary plugins cannot be run at the same time.
+
+#### Pupil Positions From Recording
+
+This plugin tries to load the pupil positions that were detected and stored during a Pupil Capture recording. It is loaded by default and is complementary to the `Offline Pupil Detector`.
+
+#### Offline Pupil Detector
+Starting with version `v9.13`, Pupil Player is able to run the same pupil detection algorithm as in Pupl Capture for recorded eye videos. This is especially important for Pupil Mobile recordings, since the app does not run pupil detection on the phone. The plugin tries to load the `eye0.mp4` and `eye1.mp4` videos in the current recording and runs the pupil detection in a separate process. `Offline Pupil Detector` is complementary to `Pupil Positions From Recording`.
+
+The `Detection Method` selector sets the detection algorithm to either `2d` or `3d` detection (see [the section on Pupil Detection](#pupil-detection) for details). The `Redetect` button restarts the detection procedure.
```

Comment:
Consider adding some use case/motivation somewhere in here like:

You can use the `Offline Pupil Detector` plugin to debug, improve, and gain insight into the pupil detection process.